File: Semester 4/Code Optimization and Debugging-II/C/Exp_05 - Sum Of Elements Using Array.c
// 5.C Program to find sum of elements using array

/*        I.            Input number of elements in array

      II.            Input the elements using array

    III.            Create a function and find the sum of array elements using loop

    IV.            Display the sum

 

 Input:

No of elements of array: 5

Case1:  10  20  30  40  50
*/

#include <stdio.h>

int sum(int arr[], int n)
{
    int sum = 0;

    for (int i = 0; i < n; i++)
    sum += arr[i];
    return sum;
}

int main()
{
    int arr[] = {10, 20, 30, 40, 50};
    int n = sizeof(arr) / sizeof(arr[0]);
    printf("%d", sum(arr, n));
    return 0;
}

/* Input: 10  20  30  40  50
   Output: 150
*/

File: Semester 4/Code Optimization and Debugging-II/Python/Exp_05 - Sum Of Elements Using Array.py
# 5.Python Program to find sum of elements using Array

# I.                    Define a function to calculate sum of array elements # using loop
#II.                  Create an array
#III.                Input number of elements and array elements using loop
#IV.                Invoke the function
# No of elements of array: 5
# Enter the array elements: 10 20 30 40 50

def result(arr,n):
    return(sum(arr))

arr = [10, 20, 30, 40, 50]
i = len(arr)
j = result(arr,i)
print (j)

# Input: 10 20 30 40 50
# Output: 150